python/tests/test_programmatically_create_an_AIConfig.py

```diff
@@ -577,17 +577,62 @@ def test_add_output_existing_prompt_no_overwrite(ai_config_runtime: AIConfigRunt
     test_result2 = ExecuteResult(
         output_type="execute_result",
         execution_count=0.0,
-        data={"role": "assistant", "content": "test output"},
+        data={"role": "assistant", "content": "test output for second time"},
         metadata={"finish_reason": "stop"},
     )
     ai_config_runtime.add_output("GreetingPrompt", test_result2)
     assert ai_config_runtime.get_latest_output("GreetingPrompt") == test_result2
 
     ai_config_runtime.delete_output("GreetingPrompt")
-
     assert ai_config_runtime.get_latest_output("GreetingPrompt") == None
 
+def test_add_output_existing_prompt_overwrite(ai_config_runtime: AIConfigRuntime):
+    """Test adding an output to an existing prompt with overwriting."""
+    base_result = ExecuteResult(
+        output_type="execute_result",
+        execution_count=0.0,
+        data={"role": "assistant", "content": "base output"},
```

Comment:
nit: We actually did a refactor recently for outputs to be text-only! Can you pls update to match that format. You can see test plans for this here:

- https://github.com/lastmile-ai/aiconfig/pull/605/files#diff-f177769a54da2bd1177e3ebd5afdec5fcfd1fa809cd63e82b193704522ba1ff0


Basically you want to change it to be in this format: 
<img width="1348" alt="Screenshot 2024-01-02 at 00 28 44" src="https://github.com/lastmile-ai/aiconfig/assets/151060367/41c4e79e-bf5e-4b85-a4b9-918eeaf2b02d">